alf/environments/gym_wrappers.py

```diff
@@ -689,3 +689,48 @@ def __init__(self, env):
     def step(self, action):
         ob, reward, done, info = self.env.step(action)
         return ob, reward, False, info
+
+
+@alf.configurable
+class DiscreteActionWrapper(gym.ActionWrapper):
```

Comment:
As we now support some native non-gym batched environment, it's better to make it a batch alf wrapper so it can also be used for those non-gym environments.

Reply:
Changed to an alf wrapper